coldfront/core/project/utils.py

```diff
@@ -162,7 +160,20 @@ def send_project_join_notification_email(project, project_user):
             'user__email', flat=True
         ))
 
-    send_email_template(subject, template_name, context, sender, receiver_list)
+    msg_plain = \
```

Comment:
Thanks, not rendering links directly is nice.

In #309/#310, we'll want to come up with a uniform way of sending HTML emails.